internal/boot/boot.go

```diff
@@ -0,0 +1,156 @@
+package boot
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/dlmiddlecote/sqlstats"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/razorpay/trino-gateway/internal/config"
+	"github.com/razorpay/trino-gateway/internal/constants/contextkeys"
+	config_reader "github.com/razorpay/trino-gateway/pkg/config"
+	"github.com/razorpay/trino-gateway/pkg/logger"
+	"github.com/razorpay/trino-gateway/pkg/spine/db"
+	"github.com/rs/xid"
+)
+
+const (
+	requestIDHttpHeaderKey = "X-Request-ID"
+	requestIDCtxKey        = "RequestID"
+)
+
+var (
+	// Config contains application configuration values.
+	Config config.Config
+
+	// DB holds the application db connection.
+	DB *db.DB
+)
+
+func init() {
+	// Init config
+	err := config_reader.NewDefaultConfig().Load(GetEnv(), &Config)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	InitLogger(context.Background())
+
+	// Init Db
+	DB, err = db.NewDb(&Config.Db)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+}
+
+func GetEnv() string {
+	// Fetch env for bootstrapping
```

Comment:
nit: move comment before the func declaration to keep the style consistent with rest of the codebase.